users/models.py

```diff
@@ -1,95 +1,99 @@
-
-    class GenderType(models.TextChoices):           #성별
-            FEMALE = 'f'
-            MALE = 'm'
-
-    # id                                                                                            #로그인테이블 ID
-    name = models.CharField(max_length=20)                                                          #이름
-    character_id = models.ForeignKey(Character, on_delete=models.SET_NULL, null=True)               #캐릭터 ID
-    height = models.FloatField(default=None, null=True)                                             #키
-    weight = models.FloatField(default=None, null=True)                                             #몸무게
-    gender = models.CharField(max_length=1, choices = GenderType.choices)                           #성별
-    is_diabetes = models.BooleanField(default=False)                                                #당뇨환자여부
-    activity = models.IntegerField(default=ActivityType.NOCHOICES, choices = ActivityType.choices)  #활동수준
-    # account_user_group_id                                                                         #그룹코드 ID
+    class ActivityType(models.IntegerChoices):  # 활동수준
+        NOCHOICES = -1  # 선택을 안 했을 때(비 당뇨인의 경우)
+        DEEPLOW = 0  # 활동이 적거나 운동을 안하는 경우
+        LOW = 1  # 가벼운 활동 및 운동
+        MIDDLE = 2  # 보통의 활동 및 운동
+        HIGH = 3  # 적극적인 활동 및 운동
+        DEEPHIGH = 4  # 매우 적극적인 활동 및 운동
+
+    class GenderType(models.TextChoices):  # 성별
+        FEMALE = 'f'
+        MALE = 'm'
+
+    id = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)  # 로그인테이블 ID
+    name = models.CharField(max_length=20)  # 이름
+    character_id = models.ForeignKey(Character, on_delete=models.SET_NULL, null=True)  # 캐릭터 ID
+    height = models.FloatField(default=None, null=True)  # 키
+    weight = models.FloatField(default=None, null=True)  # 몸무게
+    gender = models.CharField(max_length=1, choices=GenderType.choices)  # 성별
+    is_diabetes = models.BooleanField(default=False)  # 당뇨환자여부
+    activity = models.IntegerField(default=ActivityType.NOCHOICES, choices=ActivityType.choices)  # 활동수준
+    account_user_group_id = models.ForeignKey(Group, on_delete=models.SET_NULL, null=True)  # 그룹코드 ID
 
 
 # 식후 혈당량 테이블
 class BloodSugarLevel(accounts.models.BaseModel):
-    class TimelineType(models.IntegerChoices): #시간대
-        MORNING = 0         #아침
-        AFTERNOON = 1       #점심
-        NIGHT = 2           #저녁
+    class TimelineType(models.IntegerChoices):  # 시간대
+        MORNING = 0  # 아침
+        AFTERNOON = 1  # 점심
+        NIGHT = 2  # 저녁
 
-    # account_user_id
-    # diet_id
-    time = models.DateTimeField(default = None, null = True)            #혈당측정시간
-    level = models.IntegerField(max_length=4, default=None, null=True)  #혈당량
-    timeline = models.IntegerField(choices = TimelineType.choices)      #시간대
+    account_user_id = models.ForeignKey(Info, on_delete=models.SET_NULL, null=True)  # 유저 id
+    diet_id = models.ForeignKey(Data, on_delete=models.SET_NULL, null=True)  # 식단 id
```

Comment:
```suggestion
    diet_id = models.ForeignKey(diets.Data, on_delete=models.SET_NULL, null=True)  # 식단 id
```
엇 아마 다른 앱에 있는 모델에 참조하려면, 저렇게 앞에 앱 이름을 붙여줘야 하는 것으로 알고 있는데..! 그냥 모델 이름만 써도 괜찮은걸까용?